src/galois_ext.rs

```diff
@@ -0,0 +1,92 @@
+use ::poly::Polynom;
+use poly_math::*;
+
+pub struct ExtendedFieldElement {
+    rep_polynom: Polynom,
+    reduced: bool,
+}
+
+impl ExtendedFieldElement {
+    pub fn new() -> ExtendedFieldElement {
+        ExtendedFieldElement {
+            rep_polynom: polynom![0],
+            reduced: false,
+        }
+    }
+
+    pub fn reduce(&mut self) {
+        let (_, reduced_rep_polynom) = self.rep_polynom.div(&EXT_POLY!());
+        self.rep_polynom = reduced_rep_polynom;
+    }
+
+    pub fn from(original_rep_polynom: &Polynom) -> ExtendedFieldElement {
+        
```

Comment:
extraneous whitespace -- also this is better as a `From` implementations

Reply:
actually, I can see that there is also a `From` implementation and that it is equivalent to this function except for the fact that one reduces and one doesn't. What is the correct behavior? I would write to reduce to err on the side of caution.